src/site/content/en/blog/earth-webassembly/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "How we’re bringing Google Earth to the Web"
+subhead: |
+  Improving cross-browser access to Google Earth with WebAssembly.
+date: 2019-06-20
+authors:
+  - jormears
+hero: hero.jpg
+alt: A globe with code overlaid
+description: |
+  Improving cross-browser access to Google Earth with WebAssembly
+tags:
+  - post
+  - wasm
+  - WebAssembly
+  - Earth
+  - Web App
+---
+
+In an ideal world, every application that developers build, regardless of technology, would be easily made available in the browser. But there are barriers to bringing projects to the web, depending on the technology they were built with and how well that technology is supported by the various browser vendors. [WebAssembly](https://webassembly.org/) (Wasm) is a compile target standardized by the [W3C](https://www.w3.org/) that helps us solve this problem by allowing us to run codebases from languages other than JavaScript on the web.
+
+
+We’ve done just that with Google Earth, available today in [preview beta](https://g.co/earth/beta) on WebAssembly. Keep in mind that this is still a beta of Google Earth and may not be as smooth as you’re used to (try out regular [Earth for web](https://earth.google.com/web/)). You can experiment with this beta in Chrome and other Chromium-based browsers, including Edge (Canary version) and Opera, as well as Firefox. Consider this beta your inspiration if you too are looking for better cross-browser support for your native applications.
+
+
+## Why we chose WebAssembly for Google Earth
+
+We originally wrote most of Google Earth in C++ because it was initially a native application, intended for desktop install. Then we were able to port it to Android and iOS as smartphones took hold and retain most of our C++ codebase using [NDK](https://developer.android.com/ndk) and [Objective-C++](https://www.wikipedia.org/wiki/Objective-C#Objective-C++). 
+
+In 2017, when we brought Earth to the web, we used [Native Client](https://developer.chrome.com/native-client) (NaCl) to compile the C++ code and run it in Chrome browser. At the time, it was the only browser technology that allowed us to port our C++ code to the browser and give us the kind of performance Earth needed. Unfortunately, NaCl was a Chrome-only technology that never saw adoption across browsers. Now we’re starting to switch to WebAssembly, which lets us take that same code and run it across browsers. This means Earth will be available to more people across the web.
```

Comment:
Suggestion to move this first sentence, 'In 2017', to be the last part of the paragraph above, as it is still a part of the sequence of events.

Reply:
Then, start off the paragraph with 'At the time', and replace 'it' with 'NaCl': At the time, NaCl was the only.

Reply:
These two suggestions better balance the paragraphs, as well.

Reply:
Another suggestion. Instead of 'Now we're starting to switch....', say 'Now we're switching to WebAssembly. This lets us run the same code across browsers, bringing Earth to more people across the web.'

Reply:
Fixed except for the last suggestion. This wording was suggested by the Earth team since they are not actually switching over completely, just starting to. Fixed the others though.